### PR TITLE
[DC-1641] Remove `section_participation` concept

### PR DESCRIPTION
**Pull request by jp3477**
Simple suppression cleaning rule for section participation.

### CODE REVIEW

File: tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/section_participation_concept_suppression_test.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+        cls.fq_table_names = [
+            f'{project_id}.{dataset_id}.{OBSERVATION}',
+        ]
+
+        cls.fq_sandbox_table_names = [
+            f'{project_id}.{sandbox_id}.{cls.rule_instance.sandbox_table_for(OBSERVATION)}',
+        ]
+
+        cls.vocab_tables = ['concept']
```
Comment (nishanthpp93) [inline tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/section_participation_concept_suppression_test.py:56]: You might need to add `cls.vocab_tables` to `cls.fq_table_names` so those tables are cleaned after the test. Currently I believe the base cleaning test cleans `cls.fq_table_names` and `cls.fq_sandbox_table_names`.

File: tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/section_participation_concept_suppression_test.py [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+            'fields': [
+                'observation_id', 'person_id', 'observation_concept_id',
+                'observation_date', 'observation_type_concept_id'
+            ],
+            'cleaned_values': [(1, 1, 1000, self.date, 1),
+                               (2, 2, 2000, self.date, 2),
+                               (4, 4, 4000, self.date, 4)]
+        }]
+
+        self.default_test(tables_and_counts)
```
Comment (nishanthpp93) [inline tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/section_participation_concept_suppression_test.py:116]: nit: missing newline

File: data_steward/cdr_cleaner/cleaning_rules/section_participation_concept_suppression.py [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+SECTION_PARTICIPATION_CONCEPT_QUERY = JINJA_ENV.from_string("""
+CREATE OR REPLACE TABLE `{{project_id}}.{{sandbox_id}}.{{concept_suppression_lookup_table}}` AS
+SELECT c.*
+FROM `{{project_id}}.{{dataset_id}}.concept` c
+WHERE c.concept_id IN (903632)
+""")
+
+
+class SectionParticipationConceptSuppression(
+        AbstractBqLookupTableConceptSuppression):
```
Comment (nishanthpp93) [inline data_steward/cdr_cleaner/cleaning_rules/section_participation_concept_suppression.py:33]: At first I assumed only `observation_source_concept_id` would have the concept_id in observation, after looking it up in Athena. However the ticket specifies all tables so this is the intended approach and reduces work, so this is great.

**Review by nishanthpp93 — APPROVED** [review]
Looks great and achieves what is intended! Added a couple suggestions.

File: tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/section_participation_concept_suppression_test.py (hunk 1 above)
Comment (jp3477) [reply tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/section_participation_concept_suppression_test.py:56]: Will do

File: data_steward/cdr_cleaner/cleaning_rules/section_participation_concept_suppression.py (hunk 3 above)
Comment (jp3477) [reply data_steward/cdr_cleaner/cleaning_rules/section_participation_concept_suppression.py:33]: I only have observation in affected_tables. Do you think I should change it to all CDM tables then?

Comment (nishanthpp93) [reply data_steward/cdr_cleaner/cleaning_rules/section_participation_concept_suppression.py:33]: I'm not sure, let me ask on the ticket.

**Review by nishanthpp93 — APPROVED** [review]

**Pull request merged**